frontend/views/containers/proposals/Mincome.vue

```diff
@@ -92,36 +97,54 @@ export default {
   },
```

Comment:
Boyscoutting: on line 91, I think (double-check me) you can simply do:

```js
return currencies[this.groupSettings.mincomeCurrency].symbolWithCode
```

And in fact it is more correct too.

Reply:
Also, another boyscoutting issue: on line 89, we shouldn't be using a string literal `'group-setting-change'`  but the constant `PROPOSAL_GROUP_SETTING_CHANGE`

Reply:
Nice job on using the existing `groupMincomeSymbolWithCode`, but still missing fix for `'group-setting-change'`

Reply:
done